app/views/posts/index.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="row">
+  <div class="col-sm-3"></div>
+  <div class="col-sm-6">
+    <% @feed_posts.each do |post| %>
```

Comment:
[OPTIONAL]

The html / css portion of each post is the same throughout your website? If you're using it in more than one place, e.g. `posts/index.html.erb` and `users/show.html.erb`, you might want to consider putting it in a partial. The idea would be to re-use code as much as possible and avoid duplication. Imagine having to change the html in 2 or more places each time?